conformance/lang/expressions/shift-expression/left_shift.balt

```diff
@@ -0,0 +1,3816 @@
+
+function init() {
+    byte a = 112;
+    byte b = 113;
+    byte c = 12;
+    byte d = 2;
+
+    leftShiftExpr(a << b); // @output 63050394783186944
+    leftShiftExpr(a << d); // @output 448
+    leftShiftExpr(c << d); // @output 48
+    leftShiftExpr(c << b); // @output 6755399441055744
+
+    io:println((a << b).toString()); // @output 63050394783186944
+    io:println((a << d).toString()); // @output 448
+    io:println((c << d).toString()); // @output 48
+    io:println((c << b).toString()); // @output 6755399441055744
+
+    io:println((a << b).toBalString()); // @output 63050394783186944
+    io:println((a << d).toBalString()); // @output 448
+    io:println((c << d).toBalString()); // @output 48
+    io:println((c << b).toBalString()); // @output 6755399441055744
+}
+
+function leftShiftExpr(int x) {
+    io:println(x);
+}
+
+Test-Case: output
+Description: Test '<<' expression with 'byte' operands as a return statement.
+Labels: byte, int, return-stmt, shift-expr
+
+function init() {
+    byte a = 112;
+    byte b = 113;
+    byte c = 12;
+    byte d = 2;
+
+    io:println(leftShiftExpr(a, b)); // @output 63050394783186944
+    io:println(leftShiftExpr(a, d)); // @output 448
+    io:println(leftShiftExpr(c, d)); // @output 48
+    io:println(leftShiftExpr(c, b)); // @output 6755399441055744
+}
+
+function leftShiftExpr(byte x, byte y) returns int {
+    return x << y;
+}
+
+Test-Case: output
+Description: Test '<<' expression with zero and positive user-defined subtypes of integer DecimalNumbers as operands.
+Labels: DecimalNumber, shift-expr, type, union-type
+
+type Ints1 32765|21;
+type Ints2 0;
+
+function init() {
+    Ints1 a = 32765;
+    Ints1 b = 21;
+    Ints2 c = 0;
+
+    io:println(a << b); // @output 68713185280
+    io:println(a << c); // @output 32765
+    io:println(c << b); // @output 0
+    io:println(c << c); // @output 0
+}
+
+Test-Case: output
+Description: Test '<<' expression with zero and negative user-defined DecimalNumbers as operands.
+Labels: DecimalNumber, shift-expr, type, unary-minus, union-type
+
+type Ints1 -32764;
+type Ints2 -32764|0;
+
+function init() {
+    Ints1 a = -32764;
+    Ints2 b = -32764;
+    Ints2 c = 0;
+
+    io:println(a << b); // @output -524224
+    io:println(a << c); // @output -32764
+    io:println(c << b); // @output 0
+    io:println(c << c); // @output 0
+}
+
+Test-Case: output
+Description: Test '<<' expression for positive and negative user-defined DecimalNumbers as operands.
+Labels: DecimalNumber, int, shift-expr, type, unary-minus
+
+type Ints int;
+
+function init() {
+    Ints a = -111;
+    Ints b = -92;
+    Ints c = 1;
+    Ints d = 2;
+
+    io:println(a << d); // @output -444
+    io:println(c << b); // @output 68719476736
+}
+
+Test-Case: output
+Description: Test '<<' expression with user-defined subtypes of integer as the LHS operand and 'int' as the RHS
+             operand.
+Labels: DecimalNumber, int, shift-expr, type, unary-minus, union-type
```

Comment:
```suggestion
Labels: DecimalNumber, int, shift-expr, unary-minus, union-type
```

Reply:
My bad. It should be `module-type-defn`. Will update that incorrect label. Thanks.